c7n/resources/account.py

```diff
@@ -1453,44 +1440,19 @@ def get_target(self, resource):
         return resource[self.annotation]
 
 
-@actions.register('set-glue-catalog-encryption')
-class GlueDataCatalogEncryption(BaseAction):
-    """Modifies glue data catalog encryption based on specified
-    parameter
-
-    :example:
-
-    .. code-block:: yaml
-
-            policies:
-              - name: data-catalog-encryption
-                resource: account
-                filters:
-                  - type: glue-security-config
-                    CatalogEncryptionMode: DISABLED
-                actions:
-                  - type: set-glue-catalog-encryption
-                    attributes:
-                        EncryptionAtRest:
-                            CatalogEncryptionMode: SSE-KMS
-                            SseAwsKmsKeyId: alias/aws/glue
-    """
+@filters.register('glue-security-config')
+class AccountCatalogEncryptionFilter(GlueCatalogEncryptionEnabled):
+    """Filter aws account by its glue encryption status and KMS key"""
 
-    schema = type_schema(
-        'set-glue-catalog-encryption',
-        attributes={'type': 'object', "minItems": 1},
-        required=('attributes',))
+    """:example:
```

Comment:
why is this a separate string? the first string in the class is the doc string.

Reply:
ah, i did not realize this! Thanks for calling it out. i have addressed it!